pyramid_oereb/core/views/webservice.py

```diff
@@ -803,17 +802,28 @@ def get_image(self):
         Returns:
             pyramid.response.Response: Response containing the binary image content.
         """
-        method = None
-        dnr = DottedNameResolver()
-        for plr in Config.get('plrs'):
-            if str(plr.get('code')).lower() == str(self._request_.matchdict.get('theme_code')).lower():
-                method = dnr.resolve(plr.get('hooks').get('get_symbol'))
-                break
+        theme_code = self._request_.matchdict.get('theme_code')
```

Comment:
Why are we changing webservice.py in this pull request? Is it necessary for the hook method reorganization? If not, is it not an unnecessary risk, respectively, wouldn't that be better in a separate PR?

Reply:
It is related in the context at least. I try to separate the concerns. In this context I avoid passing a pyramid entity to a mehtod which has a really defined interface and therefore I reduce potential errors and rise the testability. I would vote to not split this into another PR because it means the tests need to be redone.